### DISCUSSION TITLE
Keep application interactive after test completes

### DISCUSSION THREAD

**Discussion by bukharov** [cypress-io/cypress#25177] [Questions and Help]
After we upgraded to Cypress 12 we've noticed that it now shows "Default blank page" once testing is done. In the previous versions it would keep the app interactive so it was possible to use Cypress for development as well.

Am I missing something? Some setting perhaps? Was it removed on purpose or I should report this as a bug?

**Comment by TjeuKayim** [comment]
Maybe you can do something like:

```js
  afterEach(() => {
    cy.pause();
  })
```

See https://docs.cypress.io/api/commands/pause
